core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -129,6 +129,8 @@ private TableProperties() {
   public static final String PARQUET_BATCH_SIZE = "read.parquet.vectorization.batch-size";
   public static final int PARQUET_BATCH_SIZE_DEFAULT = 5000;
 
+  public static final String ORC_BLOOM_FILTER_COLUMNS = "orc_bloom_filter_columns";
+  public static final String ORC_BLOOM_FILTER_FPP = "orc_bloom_filter_fpp";
```

Comment:
"orc.bloom.filter.fpp"

Reply:
Thank you for your review. It has been corrected according to the suggestions and the docs description has been added.